rust_dev_preview/cross_service/rest_ses/tests/api/startup.rs

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+use std::net::TcpListener;
+
+use aws_smithy_http::endpoint::Endpoint;
+use once_cell::sync::Lazy;
+use rest_ses::{
+    client::{RdsClient, SesClient},
+    configuration::{get_settings, init_environment, Environment},
+    telemetry::{get_subscriber, init_subscriber},
+};
+use wiremock::MockServer;
+
+// Ensure that the `tracing` stack is only initialised once using `once_cell`
```

Comment:
// Make sure that the `tracing` stack is only initialized once using `once_cell`.